README.md

```diff
@@ -3,9 +3,135 @@
 ConsoleDot events integrations with 3rd-party tools (such as Splunk)
 build together with [Notifications](https://github.com/RedHatInsights/notifications-backend/).
 
-## Integrations
+Currently supported integrations:
+* Splunk
+* ServiceNow
 
-* [Splunk Camel](splunk-quarkus/README.md)
+## Development
+
+### Prerequisites
+
+* podman/docker
+* [OpenJDK Development Environment](https://openjdk.java.net/guide/)
+  (`java-latest-openjdk-devel` package in Fedora)
+
+### Setup
+
+Set HTTP Event Collector (HEC) on your (local) Splunk and copy the token.
+Ensure that the HEC has SSL disabled and that all tokens are enabled.
+
+### Container Image Build
+
+```
+$ cd splunk-quarkus
+$ podman build -f Dockerfile.jvm -t quay.io/cloudservices/eventing-integrations-splunk ..
```

Comment:
Are we going to change it in the future to not have `splunk` in the name?

Reply:
Yes!